cve_bin_tool/extractor.py

```diff
@@ -26,6 +29,19 @@ class UnknownArchiveType(ValueError):
 
     # pass
 
+def run_win_cmd(cmd):
+    result = []
+    process = subprocess.Popen(cmd,
+                               shell=True,
```

Comment:
We don't want to use [`shell=True`](https://docs.python.org/3/library/subprocess.html#security-considerations)

Also, the `run_win_cmd` function looks very similar to [subprocess.check_output](https://docs.python.org/3/library/subprocess.html#subprocess.check_output) could we use that instead? or even `subprocess.call`.

Although, I'm not convinced we should be making any subprocess calls at all, I think that this is a case where we would see a performance increase by using only python libraries.